tracing-attributes/tests/fields.rs

```diff
@@ -130,17 +127,17 @@ fn parameters_with_fields() {
 
 #[test]
 fn empty_field() {
-    let span = span::mock().with_field(mock("foo").with_value(&"bar").only());
+    let span = anonymous_span().with_field(field("foo").with_value(&"bar").only());
```

Comment:
What's the thinking on calling this an `anonymous_span()`? What if it was instead `span::new().with_field(field(...))`?